### ISSUE TITLE
Allow access to private channels (groups)

### ISSUE THREAD

**Issue by jsit** [anonrig/slack-manager#22]
Would love it if slack-manager could communicate with private channels: https://api.slack.com/methods/groups.list

**Comment by anonrig** [comment]
Did you try adding the bot to private channel? @jsit

**Comment by jsit** [comment]
Yeah I did, I think private channels are "groups" and use a different API call altogether.
